django/test/runner.py

```diff
@@ -393,10 +394,11 @@ def parallel_type(value):
 
 def _init_worker(
     counter,
-    initial_settings=None,
-    serialized_contents=None,
-    process_setup=None,
-    process_setup_args=None,
+    initial_settings,
+    serialized_contents,
+    process_setup,
+    process_setup_args,
+    debug_mode,
```

Comment:
@adamchainz Do we need to change the signature here? (Can `debug_mode` not be a kwarg too?)

Reply:
In practice the args are all required (code below would crash with `None`s) so giving them defaults was just misleading.

Reply:
Yes, but it's an unrelated change for a backport to a release branch... 😬